app/views/users/show.html.erb

```diff
@@ -1 +1,16 @@
-<h1>User with the given id</h1>
\ No newline at end of file
+<h1>User with the given id</h1><div class='template'>
+<%= render partial: 'shared/user-container', locals: { user: @user } %>
+<div><%= @user.bio %></div>
+<% @user.recent_posts.each do |post| %>
+    <div>
+      <h3><%= link_to post.title, user_post_url(post) %></h3>
+      <p><%= post.text %></p>
+      <small>Comments:<%= post.comments_counter %>, Likes:<%= post.likes_counter %></small>
+    </div>
+<% end %>
+<div class='button-wrapper'>
+  <button type="button" class='see-posts-btn'>
+    <%= link_to "See all posts", user_posts_url(@user)%>
+  </button>
+</div>
+</div>
```

Comment:
- [ ] also kindly keep to the given wireframe 
your design :point_down: 
![Screenshot from 2023-08-31 20-31-01](https://github.com/Enoisong/blog_app/assets/66097371/8a3a2e83-ca6b-4349-881e-12a8272c041a)
design guide :point_down: 
![Screenshot from 2023-07-20 22-23-13](https://github.com/Enoisong/blog_app/assets/66097371/9aa92755-a33e-44a4-b60b-1a46cbd2e8f6)